deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts

```diff
@@ -223,12 +229,89 @@ export class MigrationServiceCore extends Stack {
             enableExecuteCommand: true,
             securityGroups: props.securityGroups,
             vpcSubnets: props.vpc.selectSubnets({subnetType: SubnetType.PRIVATE_WITH_EGRESS}),
-            cloudMapOptions: cloudMapOptions
+            cloudMapOptions: cloudMapOptions,
         });
+        
+        if (props.targetGroups) {
+            props.targetGroups.forEach(tg => tg.addTarget(fargateService));
+        }
 
         if (props.serviceDiscoveryEnabled) {
             this.addServiceDiscoveryRecords(fargateService, props.serviceDiscoveryPort)
         }
     }
 
-}
\ No newline at end of file
+    getSecureListenerSslPolicy() {
```

Comment:
These load balancer utilities seem like they belong to a utilities class

Reply:
Moved to network stack